packages/hdwallet-gridplus/src/bitcoin.ts

```
@@ -0,0 +1,627 @@
+export async function btcSignTx(client: Client, msg: core.BTCSignTx): Promise<core.BTCSignedTx | null> {
+  // All UTXOs (Bitcoin, Dogecoin, Litecoin, Bitcoin Cash) use Bitcoin-compatible transaction formats.
+  // The 'BTC' currency parameter is just SDK routing - the device signs Bitcoin-formatted transactions.
+  // Address derivation already handles all coins via client-side derivation with proper network parameters.
+
+  // Calculate fee: total inputs - total outputs
+  const totalInputValue = msg.inputs.reduce((sum, input) => sum + parseInt(input.amount || "0"), 0);
+  const totalOutputValue = msg.outputs.reduce((sum, output) => sum + parseInt(output.amount || "0"), 0);
+  const fee = totalInputValue - totalOutputValue;
+
+  // Find change output and its path
+  const changeOutput = msg.outputs.find(o => o.isChange);
```

ESLint

[error] Replace `o` with `(o)`

(prettier/prettier)